src/usr/local/www/pfblockerng/pfblockerng_software.php

```
@@ -85,15 +84,29 @@
 	pfb_software_check_config_write((pfb_filter($_POST['pfb_software_check'] ?? '', PFB_FILTER_ON_OFF, 'software') ?: '') === 'on');
+	$pfb_ca_was_consented = pfb_pkg_ca_consent_enabled();
+	$pfb_ca_token = pfb_pkgconf_ca_save($_POST);
 	write_config('[pfBlockerNG] save Software settings');
-	header('Location: /pfblockerng/pfblockerng_software.php');
-	exit;
+	$pfb_ca_ok = pfb_pkgconf_ca_apply($pfb_ca_token, $pfb_ca_was_consented);
+	if ($pfb_ca_ok) {
+		header('Location: /pfblockerng/pfblockerng_software.php');
+		exit;
+	}
+	$input_errors[] = 'The setting was saved, but pfBlockerNG could not update ' . PFB_PKG_CONF
```

PHPStan (2.2.7)

[error] Constant PFB_PKG_CONF not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)